apps/finance/test/finance.js

```diff
@@ -32,6 +32,9 @@ const tokenTestGroups = [
     },
 ]
 
+// Hack to make contract-test-helpers work with Web3 1.x.
+web3.BigNumber = web3.utils.BN
```

Comment:
Interesting... where do we need this in contract-helpers?

If we do intend to keep this, let's add a big TODO that we can fix immediately.